utils/manifests/utils.go

```diff
@@ -156,70 +122,6 @@ func RemoveHelmTemplatingFromCRD(crdyaml *string) {
 	*crdyaml = strings.Join(yamlArr, "\n---\n")
 }
 
-func getCrdnames(s string) []string {
```

Comment:
@MUzairS15, will you confirm that Meshery Server/other doesn't use this function or the filterYaml() function any longer?

Reply:
@KiptoonKipkurui you might be able to offer similar confirmation

Reply:
Confirmed: No use inside of Meshery Server.

Reply:
Good check